app/upgrades.go

```diff
@@ -21,9 +21,17 @@ func (app *App) RegisterUpgradeHandlers(cfg module.Configurator) {
 		if err != nil {
 			return newVM, err
 		}
-		params := app.WasmKeeper.GetParams(ctx)
-		params.CodeUploadAccess = wasmtypes.AllowNobody
-		app.WasmKeeper.SetParams(ctx, params)
+		// consensus params
+		// increase max gas as part of the upgrade to handle cosmwam
+		consensusParams := app.BaseApp.GetConsensusParams(ctx)
+		consensusParams.Block.MaxGas = 75_000_000 // 75M
```

Comment:
`75_000_000` listed here, but `genParams.ConsensusParams.Block.MaxGas 100_000_000` is the difference intentional?

Reply:
`app/upgrades.go` goes into the actual mainnet upgrade, the genesis  command is just to bootstrap new networks (local development, devnet etc) its unrelated.

Reply:
maybe we could make testnet, devnet use less gas than mainnet so there is no unexpected behavior going from devnet to main net?

Reply:
I can address that on a different PR, for local development I think is fine because you don't have to optimize contracts and can iterate fast. But agree on devnet to make it the same